app.py

```diff
@@ -98,26 +97,83 @@ def find_font(name):
 videos = get_videos(searchPath)
 video_names = get_video_names(searchPath)
 
-# Preload the response; it's always the same
-subs = subs2json(load_subs(videos))
+# Loads the video subtitles
+video_subs = load_subs(videos)
+# Creates a list of video data with subtitles
+videos_with_subs = [{ 'title': video_names[idx], 'id': idx, 'subs': v[0] } for idx, v in enumerate(video_subs)]
+# Convert sub data to JSON data
+subs = subs2json(video_subs)
+# Creates a list of the installed fonts
+fonts = font_manager.findSystemFonts(fontpaths=None, fontext='ttf')
```

Comment:
Though it shouldn't be impossible, working with seconds in the UI would feel annoying since you will have to deal with floating numbers

I would argue a user does care about the font that is used. Perhaps you want to just use a standard subtitle font, or the typical meme impact font